components/kubernetes/kind.go

```diff
@@ -27,10 +27,10 @@ const (
 var kindClusterConfig string
 
 // Install Kind on a Linux virtual machine.
-func NewKindCluster(env config.Env, vm *remote.Host, resourceName, kindClusterName string, kubeVersion string, opts ...pulumi.ResourceOption) (*Cluster, error) {
-	return components.NewComponent(env, resourceName, func(clusterComp *Cluster) error {
+func NewKindCluster(env config.Env, vm *remote.Host, name string, kubeVersion string, opts ...pulumi.ResourceOption) (*Cluster, error) {
+	return components.NewComponent(env, name, func(clusterComp *Cluster) error {
+		kindClusterName := env.CommonNamer().DisplayName(49) // We can have some issues if the name is longer than 50 characters
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
What is the resulting name for the cluster, using common namer?

Reply:
The resulting name will be the name of the stack, with a limit of 49 characters here. We have something similar for the naming of the EKS cluster: https://github.com/DataDog/test-infra-definitions/blob/main/scenarios/aws/eks/cluster.go#L88